src/components/pushClient/Banner.jsx

```diff
@@ -59,7 +59,7 @@ class BannerClient extends Component {
             this.markAsSeen('banner')
           }}
         >
-          <span>{t('Nav.btn-client-mobile')}</span>
+          {t('Nav.btn-client-mobile')}
```

Comment:
Shouldn't it be in a `label` props ?

Reply:
Well now, with version 7.7.2, yes ! Before that the `label` prop for `ButtonLinks` was broken. 👍